validator/src/main/resources/expected-data-template/otlpToAMPExpectedMetric.mustache

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "metric": {
```

Comment:
This has to do with when there is one metric it will set the quantile as 0 and 1. This is really 1 metric